notify/incidentio/incidentio_test.go

```diff
@@ -0,0 +1,303 @@
+// Copyright 2025 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package incidentio
```

Comment:
Hi @rorymalcolm 
Thanks for working on the PR and I hope it will be merged soon!

Small nit / idea to consider: I noticed that PagerDuty is able to truncate to too large payloads to make sure it will fit into the the limits.
It prints message like this:
```
Truncated Details because message of size 1MB876KB984B exceeds limit 512KB
```
While I believe alert payload should not exceed the size, it might be worth adding a safeguard to make sure alert will be delivered even if it is too bit and must be truncated.

Reply:
> Hi @rorymalcolm 
> 
> Thanks for working on the PR and I hope it will be merged soon!
> 
> 
> 
> Small nit / idea to consider: I noticed that PagerDuty is able to truncate too large payloads to make sure it will fit into the limits.
> 
> It prints message like this:
> 
> ```
> 
> Truncated Details because message of size 1MB876KB984B exceeds limit 512KB
> 
> ```
> 
> While I believe alert payload should not exceed the size, it might be worth adding a safeguard to make sure alert will be delivered even if it is too bit and must be truncated.

Hi Nikita 👋 

This makes sense! I'll look to add it

Reply:
Done!